homeassistant/components/onkyo/quality_scale.yaml

```
@@ -0,0 +1,89 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration uses a push API. No polling required.
+  brands: done
+  common-modules: done
+  config-flow:
+    status: todo
+    comment: |
+      The data_descriptions are missing.
+  config-flow-test-coverage:
+    status: todo
+    comment: |
+      Coverage is 100%, but the tests need to be improved.
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration do not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: todo
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: todo
+  integration-owner: done
+  log-when-unavailable: todo
+  parallel-updates: todo
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      This integration does not require authentication.
+  test-coverage: todo
+  # Gold
+  devices: todo
+  diagnostics: todo
+  discovery: todo
+  discovery-update-info: todo
+  docs-data-update: todo
+  docs-examples: todo
+  docs-known-limitations: done
```

I haven't spotted such a section in our documentation.

```suggestion
  docs-known-limitations: todo
```

---

The only limitation that I know of, is the requirement to enable "Network Standby" in the receiver (so that we can turn it back on after turning it off, for example). This limitation is mentioned at the very top of the documentation.

If you or anyone else have any ideas as to what might constitute a limitation in the context of this integration, I obviously welcome all suggestions and feedback. I just couldn't think of anything else myself.

---

I think you can mention that we can't fetch the names of the outputs ourselves and that you have to set them yourselves. Otherwise its still good to be explicit and tell that there is no known limitation